app/assets/src/components/views/report/SampleDetailsSidebar/NotesTab.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import MetadataSection from "./MetadataSection";
+import Textarea from "~/components/ui/controls/Textarea";
+import PropTypes from "prop-types";
+import cs from "./sample_details_sidebar.scss";
+
+class NotesTab extends React.Component {
+  state = {
+    editing: false
+  };
+
+  toggleEditing = () => {
+    this.setState({
+      editing: !this.state.editing
+    });
+  };
+
+  render() {
+    const { notes, onNoteChange, onNoteSave, savePending } = this.props;
+
+    const notesEmpty = !notes || notes.length === 0;
+
+    return (
+      <MetadataSection
+        editable
+        onEditToggle={() => this.toggleEditing()}
+        editing={this.state.editing}
+        title="Notes"
+        savePending={savePending}
+        alwaysShowEditLink={notesEmpty}
```

Comment:
re alwaysShowEditLink: interesting, did Design ask for this? I would've been OK with it being on-hover like the others but it's nice